src/main/webapp/style.css

```diff
@@ -69,3 +65,39 @@ input[type=submit]:hover{
 	display: inline-block; 
 	vertical-align: middle;  
 }
+
+
+/* Animation and Background colors */
+.background {
+  animation:slide 3s ease-in-out infinite alternate;
+  background-image: linear-gradient(-60deg, darkgrey 50%, black
+    50%);
+  bottom:0;
+  left:-50%;
+  opacity:.5;
+  position:fixed;
+  right:-50%;
+  top:0;
+  z-index:-1;
+}
+.background2 {
+  animation-direction:alternate-reverse;
+  animation-duration:7s;
+}
+.background3 {
+  animation-duration:8s;
+}
+
+
+h1 {
+  font-family:monospace;
+}
```

Comment:
nit: add one and only one new line in between the elements to be consistent?

Reply:
on it 👍